systems-ui/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import Header from "./header/Header";
+
+class App extends Component {
```

Comment:
When you say stateless, you mean the arrow function component? ```(props) => <div>a</div>```

Yeah it could be :) I really like the new way of defining stateless components, much more readable and less verbose. There is a change this component could evolve to a stateful component in the future, but for that we could use the https://github.com/acdlite/recompose and still keep the arrow function approach here 👍